dhis-2/dhis-services/dhis-service-field-filtering/src/main/java/org/hisp/dhis/fieldfiltering/FieldFilterService.java

```diff
@@ -129,6 +121,29 @@ private ObjectMapper configureFieldFilterObjectMapper( ObjectMapper objectMapper
         return objectNodes;
     }
 
+    public ObjectNode createObjectNode()
```

Comment:
`private`?

Reply:
I'll probably move them at some point, but right now these are utilty methods used outside of this class for constructing JSON objects and arrays.